src/components/ec-smart-table/ec-smart-table.spec.js

```diff
@@ -677,4 +677,61 @@ describe('EcSmartTable', () => {
       }]);
     });
   });
+
+  describe('infiniteScroll', () => {
+    it('should render the element "tableEndDetector" if "isInfiniteScrollEnabled" prop is true', () => {
+      const wrapper = mountEcSmartTableWithData(data, {
+        isInfiniteScrollEnabled: true,
+      });
+      const tableEndDetector = wrapper.findByDataTest('ec-smart-table__table-end-detector');
+      expect(tableEndDetector.element).toMatchSnapshot();
+    });
+
+    it('should not render EcLoading loading SVG if "isInfiniteScrollEnabled" prop is true', () => {
+      const wrapper = mountEcSmartTableWithData(data, {
+        isInfiniteScrollEnabled: true,
```

Comment:
loading is rendered when `isFetching` prop is true. So we should set it to true and check whether loading is rendered.

Reply:
Have you tried to fail this test by setting `isInfiniteScrollEnabled: false`?

Reply:
eh... no, I didn't, thanks for noticing!